src/lang/en-US.ts

```diff
@@ -58,6 +63,15 @@ const CommonMessages: ResolvedIntlConfig['messages'] = {
     'capturesource.fetch.failed': `Failed to fetch source types`,
     'username.label': `User Name`,
 
+    // Login Page Stuff
+    'login.main.message': `When running locally you can login with whatever name you want.`,
+
+    // Registration Page
+    'register.heading': `We're currently accepting Beta partners.`,
+    'register.main.message': `Please enter your information and our team will approve your account.`,
+    'register.documentAcknowledgement': `Accept our Terms of Service and Privacy Policy`,
+    'register.existingAccount': `Already have an account? `,
```

Comment:
I think if we need a space after something it should not be handled in the content. We can force it with a `{' '}` in the JSX.

Reply:
**PENDING:** I went back and forth on this because `{' '}` felt awkward to me, however it is the most reasonable way to ensure the trailing space is always present. Oversight. Will update.